### PR TITLE
BZ-1328722: [ALL LANG] Word 'of' is unlocalized in each string 'x-x of x' under each table result list.

### PR DESCRIPTION
**Pull request by paulovmr**

### CODE REVIEW

**Comment by manstis** [comment]
@paulovmr Super * thanks!

+1 to merge.

File: uberfire-widgets/uberfire-widgets-table/src/main/java/org/uberfire/ext/widgets/table/client/UberfireSimplePager.java [hunk 1]
```diff
@@ -372,16 +373,16 @@ protected String createText() {
...
-                    + " of "
+                    + " " + of() + " "
                     + formatter.format( dataSize );
         }
         return formatter.format( pageStart )
                 + "-"
                 + formatter.format( endIndex )
-                + ( exact ? " of " : " of over " )
+                + ( exact ? " " + of() + " " : " " + of() + " over " )
                 + formatter.format( dataSize );
```
Comment (csadilek) [inline uberfire-widgets/uberfire-widgets-table/src/main/java/org/uberfire/ext/widgets/table/client/UberfireSimplePager.java]: @paulovmr @manstis shouldn't over be localized here as well?

Comment (manstis) [inline uberfire-widgets/uberfire-widgets-table/src/main/java/org/uberfire/ext/widgets/table/client/UberfireSimplePager.java]: @csadilek Yes, well spotted.. although the BZ only says "of" should be localized, I think we should show willing ;)

@paulovmr Sorry.. I missed that.

Comment (paulovmr) [inline uberfire-widgets/uberfire-widgets-table/src/main/java/org/uberfire/ext/widgets/table/client/UberfireSimplePager.java]: @csadilek Nice catch, I missed that too...

Comment (paulovmr) [inline uberfire-widgets/uberfire-widgets-table/src/main/java/org/uberfire/ext/widgets/table/client/UberfireSimplePager.java]: @manstis No problem :)

**Comment by paulovmr** [comment]
@csadilek @manstis Ok, ready for review. Thanks!

**Comment by csadilek** [comment]
+1

**Pull request merged**